tests/test_pytesseract.py

```diff
@@ -1,25 +1,239 @@
+# encoding: utf-8
+
 import os.path
+from sys import platform, version_info
+from multiprocessing import Pool
 
 import pytest
 
-from pytesseract import image_to_string
+from pytesseract.pytesseract import get_tesseract_version, image_to_data, \
```

Comment:
We should import from as much as possible from pytesseract instead from pytesseract.pytesseract in order to detect user import problems. A.k.a if we change some imports in the __init__ file, we should be able to detect it here if we import from pytesseract. So maybe it is good idea to change all pytesseract.pytesseract to pytesseract for all exported objects and let pytesseract.pytesseract for everything else that is not exported.

Reply:
Thanks! Yes you're right. I changed and [sorted](https://github.com/timothycrosley/isort) the imports.